### ISSUE TITLE
Not Recognising the Password (I gave my own password and tried on myself)

### ISSUE THREAD

**Issue by HackEthos4u** [4lxprime/instabrute#4]
Tool not recognising the password, I tried giving my own password but not shown any success or this password is right like that if you able to **fix this issue it really helpful**

**Comment by 4lxprime** [comment]
yes, it's depreciated, but I don't think I'm going to try to correct it because with my studies and personal projects I don't have the time for that.

**Comment by sonowalprasanna9-png** [comment]
kanyaa._p this account password
